crates/nargo_cli/tests/compile_failure/divide_by_zero/src/main.nr

```diff
@@ -0,0 +1,6 @@
+use dep::std;
+
+fn main() {
```

Comment:
can you add a test case where the numerator is not a constant, so a parameter to main, and then one where both the numerator and denominator are non constant. This should fail execution

Reply:
sure thing